composer.json

```diff
@@ -31,8 +31,9 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "rokka/client": "^1.2.0",
+        "rokka/client": "^1.3.1",
         "rokka/client-cli": "^1.6.1",
+        "rokka/twig": "dev-develop as 0.0.2-dev",
```

Comment:
i got installed 0.0.3. should we require `0.0.3@dev`? or test a bit more and then release 1.0.0 of the twig thing?

Reply:
i changed that to ^0.0.3 for now

Reply:
would be great, if someone would test it a little bit in a real world project, then I'm happy to release 1.0.0 of the twig bundle and merge this. OTOH, we can also just be adventurous and release 1.0.0 anyway ;)

Reply:
done